__tests__/app.test.js

```diff
@@ -234,4 +234,90 @@ describe("/api", () => {
         });
     });
   });
+  describe("PATCH /articles/:article_id", () => {
+    test("PATCH 200: Should increment the votes and respond with updated article array", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({
+          inc_votes: 1,
+        })
+        .expect(200)
+        .then(({body}) => {
+          expect(typeof body.votes).toBe('number');
+          expect(body.votes).toEqual(101);
+        });
+    });
+
+    test("PATCH 200: Should increment the votes and respond with updated article array", () => {
```

Comment:
We have two tests with the same description here, are we checking a different behaviour? If so the description should reflect that.